uast/iter.go

```diff
@@ -0,0 +1,97 @@
+package uast
+
+import (
+	"sort"
+
+	"gopkg.in/bblfsh/sdk.v2/uast/nodes"
+)
+
+// NewPositionalIterator creates a new iterator that enumerates all object nodes, sorting them by positions in the source file.
+// Nodes with no positions will be enumerated last.
+func NewPositionalIterator(root nodes.External) nodes.Iterator {
+	return &posIterator{root: root}
+}
+
+type posIterator struct {
```

Comment:
Should be positionIterator to be in line with the other rename and avoid being easy to mistype as postIterator (or vice-versa).

Reply:
Mmm... It's an unexported type, so it doesn't matter that much. Personally (and following Go convention), I don't want long names in the method's receiver.